src/kubectl.ts

```diff
@@ -139,7 +139,7 @@ async function invokeAsync(context: Context, command: string): Promise<ShellResu
 async function invokeAsyncWithProgress(context: Context, command: string, progressMessage: string): Promise<ShellResult> {
     return context.host.withProgress(async (p) => {
         p.report({ message: progressMessage });
-        return await invokeAsync(context, command);
+        return await invokeAsync(context, command, null);
```

Comment:
If we make `stdin` optional in `invokeAsync` then we don't need to modify this and other invocations.  I don't feel that the explicit passing of `null` provides any clarity advantage over just leaving it out, but open to disagreement!

Reply:
removed.